ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -117,6 +117,51 @@ tests:
           cpu: 300m
           memory: 500Mi
     workflow: generic-claim
+- as: e2e-aws-ocp-4.8-periodic
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.8"
+  cron: 0 */24 * * *
```

Comment:
Is there a reason why you use `*/24`? How about `0 4 * * *`?
`@daily` would be even better if it is supported. Unfortunately the [reference docs](https://steps.ci.openshift.org/ci-operator-reference) don't say that.

Reply:
> Is there a reason why you use */24? How about 0 4 * * *?

What is the difference between the 2?

I see that assisted is using `'@daily` https://github.com/ybettan/release/blob/98baa4c0db1e828de7439c118e83883904714e98/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml#L290